pkg/scanners/trivy/trivy.go

```diff
@@ -160,12 +160,11 @@ func initScanner(userConfig *Config) (Scanner, error) {
 	sugar := logger.Sugar()
 	trivylogger.Logger = sugar
 
-	runtime := os.Getenv(utils.EnvEraserContainerRuntime)
-	if runtime == "" {
-		runtime = utils.RuntimeContainerd
+	userConfig.Runtime = unversioned.RuntimeSpec{
+		Name:    unversioned.Runtime(utils.EnvEraserRuntimeName),
```

Comment:
This sets `userConfig.Runtime.Name` name to `ERASER_RUNTIME_NAME` ! It leads to the question of how this is working at all.